Packs/JoeSecurity/Integrations/JoeSecurityV2/JoeSecurityV2.py

```diff
@@ -898,6 +898,8 @@ def main() -> None:  # pragma: no cover
     """
     api_key = demisto.get(demisto.params(), 'credentials.password')
     base_url = demisto.params().get('url')
+    if 'api' not in base_url:
```

Comment:
Add a debug log write here printing api was not found in base_url adding it and how the base url looks after adding